playbooks/openstack/sample-inventory/group_vars/all.yml

```diff
@@ -106,6 +106,16 @@ openshift_openstack_num_infra: 1
 openshift_openstack_num_cns: 0
 openshift_openstack_num_nodes: 2
 
+# # Public IP Allocation
+# # - manage which node roles are allocated public IP addresses
+# # - by default, all roles are given Public IP addresses
+#openshift_openstack_master_floating_ip: true
+#openshift_openstack_infra_floating_ip: true
+#openshift_openstack_app_floating_ip: true
+#openshift_openstack_etcd_floating_ip: false
+#openshift_openstack_load_balencer_floating_ip: false
```

Comment:
Typo: this should say `balAncer` not `balEcer`. Needs to be fixed everywhere in this patch.